austation/code/game/objects/structures/party_palloons.dm

```diff
@@ -0,0 +1,82 @@
+/////////////////////  CONSUMABLE BALLOON  /////////////////////
+
+
+/obj/item/balloon
+	name = "deflated balloon"
+	desc = "A small, deflated balloon.  You can use it in your hand to inflate it."
+	icon = 'austation/icons/obj/structures/balloons.dmi'
+	icon_state = "red"
+	w_class = WEIGHT_CLASS_TINY
+	var/coloring = "red"
+	var/inflate_time = 20
+
+/obj/item/balloon/Initialize()
+	. = ..()
+	var/list/colorslist = list(
+		"red",
+		"blue",
+		"green",
+		"yellow",
+		"purple",
+		"pink"
+	)
```

Comment:
This should be a static object variable rather than something that's made on initialize

Reply:
changed